sdks/builds/Makefile

```diff
@@ -14,6 +14,11 @@ UNAME=$(shell uname)
 ifneq ($(UNAME),Darwin)
 # iOS requires Xcode to be available, and Xcode is only available on macOS
 DISABLE_IOS=1
+
+ifeq ($(UNAME),Linux)
+LINUX_FLAVOR=$(shell ./determine-linux-flavor.sh)
```

Comment:
Since this is only useful for `mxe`, put it in `mxe.mk` instead.

Reply:
For now, but would it be bad to keep it in the top level file where it can be exported for all/any other bits which may need special treatment?

Reply:
We can always move it back here if it's needed.